admin/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="main.js"></script>
+    <script>
+        window.addEventListener('load', main);
+    </script>
+    <title>Admin | SimpleApp</title>
+</head>
+<body>
+    <p>This is a basic test</p>
+</body>
+</html>
```

Comment:
Missing new line at the end of the file

Reply:
Done. Why is this important?

Reply:
It's good programming practice. The source of some issues and all linters complain. Also see:

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline